app/src/sharedTest/java/org/oppia/android/app/profile/AdminAuthActivityTest.kt

```diff
@@ -610,6 +618,42 @@ class AdminAuthActivityTest {
     }
   }
 
+  @Test
+  fun testAdminAuthActivity_hasCorrectActivityLabel_adminControls() {
+    activityTestRule.launchActivity(
+      AdminAuthActivity.createAdminAuthActivityIntent(
+        context = context,
+        adminPin = "12345",
+        profileId = internalProfileId,
+        colorRgb = -10710042,
+        adminPinEnum = AdminAuthEnum.PROFILE_ADMIN_CONTROLS.value
+      )
+    )
+    val title = activityTestRule.activity.title
+
+    // Verify that the activity label is correct as a proxy to verify TalkBack will announce the
+    // correct string when it's read out.
+    Truth.assertThat(title).isEqualTo(context.getString(R.string.admin_auth_controls_label))
```

Comment:
As we are having a complete import of truth, we can skip using truth here. 
`import com.google.common.truth.Truth`

```kotlin
assertThat(title).isEqualTo(context.getString(R.string.admin_auth_controls_label))
```

Reply:
ditto below

Reply:
+1